lib/scan_code/scan.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'package:barcode_scan2/barcode_scan2.dart';
+import 'package:flutter/services.dart';
+import 'package:generator/widgets/appbar.dart';
+
+class ScanPage extends StatefulWidget {
+  @override
+  _ScanPageState createState() => _ScanPageState();
+}
+
+class _ScanPageState extends State<ScanPage> {
+  @override
+  void initState() {
+    super.initState();
+    _scan();
+  }
+
+  String qrCodeResult = "";
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: backAppBar(context, label: "SCAN"),
+      body: SafeArea(child: scanWidget()),
+    );
+  }
+
+  Widget scanWidget() => Center(
+        child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.center,
```

Comment:
**suggestion (nit)**: `CrossAxisAlignment.center` is the default, so no need to pass it

Reply:
Thanks